src/test/java/com/jcabi/aspects/ImmutableTest.java

```diff
@@ -89,6 +91,35 @@ public void catchesTypesMutableByClassInheritance() {
     }
 
     /**
+     * Test Version.
+     */
+    @Test
+    public void testVersion() {
+        try {
+            new Mutable();
+        } catch (final IllegalStateException ex) {
+            MatcherAssert.assertThat(
+                ex.getMessage(),
+                Matchers.not(
+                    Matchers.containsString(
+                        "${projectVersion}"
```

Comment:
@pecko @pinaf It should instead assert that `ex.getMessage()` contains `Version.CURRENT.buildNumber()` and `Version.CURRENT.projectVersion()`.